Makefile

```diff
@@ -45,7 +45,7 @@ help:
 
 .PHONY: virtualenv
 virtualenv: ## create virtual environment typically used for development purposes
-	virtualenv -p python3.9 --always-copy --prompt='(twindb_backup)' ~/.virtualenvs/twindb-backup/
+	virtualenv -p python3 --always-copy --prompt='(twindb_backup)' ~/.virtualenvs/twindb-backup/
```

Comment:
Python minor version is different on different OSs, so I just use puython3. It's important no supported OSs use deprecated 3.6 anymore.